healvis/observatory.py

```diff
@@ -261,6 +300,9 @@ def make_visibilities(self, shell, Nprocs=1, beam_pol='pI'):
         conv_fact = jy2Tsr(np.array(self.freqs), bm=pix_area_sr)
         self.Ntimes = len(self.pointing_centers)
 
+        if self.pointing_centers is None:
```

Comment:
would it be possible for this method just to call `set_pointings()` itself?

Reply:
Only if it is given a set of times. I've added that as an optional keyword and allowed this method to call `set_pointings`.